README.md

```diff
@@ -154,25 +127,12 @@ To use this application, follow these steps:
 
 To run tests, run the following command:
```

Comment:
- [OPTIONAL] You have duplicated your README file's `Contributions` section. Kindly remove one 🚮 

![image](https://user-images.githubusercontent.com/79058364/230603686-dbff0614-9496-40af-b4de-d8d5c94b9434.png)

Reply:
- [OPTIONAL] Remove this from your README file's Contributions section 👇🏻 This setion is suppoert to onatain a link to the issue page and not the instruction on how to install and clone your app

![11](https://user-images.githubusercontent.com/79058364/230604037-f8407830-5875-439b-9167-ee0191f61944.png)